src/Modals.tsx

```diff
@@ -28,10 +28,10 @@ export const Modals = () => {
     isSignatureModalOpen,
     isNotificationPwaModalOpen
   } = useModals()
-  const { close: closeWeb3Modal } = useWeb3Modal()
-  const { open: isWeb3ModalOpen } = useWeb3ModalState()
+  useWeb3Modal()
```

Comment:
If we don't get any date from the w3m hooks, do we need to use them? I think they don't have any life cycle methods right?